### PR TITLE
Harness for Go `crypto/x509`

### LINKED ISSUE #9: Add an example harness against a known-good library (go crypto/x509 probably)
This will help validate that testcases we write are correct. Or we'll find bugs in go. Either way.

### PR DESCRIPTION
**Pull request by tnytown**
Closes #9.

### CODE REVIEW

File: harness/gocryptox509/Makefile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+.PHONY: all
+all:
```
Comment (tnytown) [inline harness/gocryptox509/Makefile:2]: Do we need a target that runs the harness in the toplevel Makefile?

File: .envrc [hunk 2]
```diff
@@ -0,0 +1,6 @@
+source_up_if_exists
+
+export VIRTUAL_ENV=env
+layout python
+
+use flake
```
Comment (woodruffw) [inline .envrc]: Local dev state? 🙂

File: harness/gocryptox509/Makefile (hunk 1 above)
Comment (woodruffw) [reply harness/gocryptox509/Makefile:2]: Yeah, that'd be good IMO -- let's do a PHONY target like `make test-go` for the time being.

File: .envrc (hunk 2 above)
Comment (tnytown) [reply .envrc]: oops, forgot to stash nix stuff

**Review by woodruffw — APPROVED** [review]
LGTM; harness failure is fine for now.

**Pull request merged**